app/style/components/mention-suggestions.less

```diff
@@ -17,7 +17,62 @@
  *
  */
 
+.conversation-input-bar-mention-suggestion {
+  position: fixed;
+  z-index: @z-index-context;
+  border: 0 solid rgba(151, 151, 151, 0.24);
+  background-color: #fff;
+  border-radius: 8px;
+  box-shadow: 0 8px 30px 0 rgba(0, 0, 0, 0.08);
+}
+
 .mention-suggestion-list {
   display: flex;
   flex-direction: column-reverse;
+  padding: 4px 0;
+
+  &__item {
+    display: flex;
+    height: 32px;
+    align-items: center;
+    padding: 0 12px;
+
+    &--highlighted {
+      background-color: fade(@graphite-dark, 4%);
+    }
+
+    participant-avatar {
+      margin-right: 12px;
+    }
+
+    &__name {
+      margin-right: 4px;
+      font-size: 14px;
+      line-height: 14px;
+    }
+
+    &__username {
+      margin: 0 4px;
+      color: fade(@graphite-dark, 40%);
+      font-size: 12px;
+      line-height: 14px;
+
+      &::before {
+        display: inline;
+        content: '@';
+      }
+    }
+
+    &__guest-badge {
+      margin-left: 12px;
+
+      svg {
+        width: 12px;
+        height: 12px;
+        path {
+          fill: fade(@graphite-dark, 40%);
+        }
+      }
+    }
+  }
```

Comment:
I am missing the full width definition with right margin of the complete list that is mentioned in the spec.

Reply:
We are not doing the full width anymore. And this should be reflected in the specs.

Reply:
So the width will jump depending on which suggestions get shown in the list?

Reply:
Yes, we will see how that works out and might introduce a maximum width.